File: LeetCode-Problems/Easy/Count-Number-of-Even-Lengthed-Integers.java
//Java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for (int i = 0; i < nums.length; i++){
            int numLength = Integer.toString(nums[i]).length();
            if (numLength % 2 == 0){
                count++;
            }
        }
        return count;
    }
}

File: LeetCode-Problems/Easy/Count-Number-of-Even-Lengthed-Integers.py
# Python
class Solution(object):
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        for x in range (0, len(nums)):
            numLength = len(str(nums[x]))
            if (numLength % 2 == 0):
                count = count + 1
        return count
